### DISCUSSION TITLE
에디터에서 스크립트 자동완성이 되질 않습니다.

### DISCUSSION THREAD

**Discussion by smartbh** [naverz/zepeto-studio-kor#17] [5. Q&A]
![image](https://user-images.githubusercontent.com/43138813/144077526-74fafe3a-9b57-403b-b547-dbad61f5b7c1.png)

일반적으로는 타입스크립트나 C#코드 같은경우

스크립트 자동완성 기능이 있는거로 알고 있습니다.

하지만 저는 자동완성이 안되고 입력이 안되는것처럼

기본 함수 색으로 나와서 인식이 된건지 아닌지 혼란을 주는데

오타를 내지않고 그대로 받아 적으면 실행은 되서 더 혼란스럽습니다.

타입스크립트에서도 자동완성이 될 수 있게 하는 방법이 없을까요?

OnTriggerEnter와 같은 함수들도 자동완성이 되지 않습니다.

**Comment by x4cf** [comment]
온트리거 함수는 자동완성이 되지 않는데 맞고, GameObject 라고 적으신 매게변수는 자료형에 따라 자동완성이 노출됩니다. 
const _gameObject: GameObject = GameObject; 와 같이 코드를 작성하시면 _gameObject. 의 자동완성 기능을 이용하실 수 있습니다.

**Comment by NoahHahm** [comment]
IDE 환경의 정보를 혹시 알려주실수 있을까요?

**Reply by smartbh** [reply]
비주얼스튜디오 2019 입니다

**Reply by NoahHahm** [reply]
지금은 VS Code / Rider 두개의 IDE 만 공식 지원하고 있습니다.
비주얼스튜디오 2019는 지원할수 있도록 내부에 전달하겠습니다.
